app/views/users/new.html.erb

```diff
@@ -1,22 +1,89 @@
-<div class="flex flex-col items-center justify-center w-full h-full space-y-4">
-  <div class="p-3 bg-white border border-gray-100 rounded-full">
-    <%= image_tag("logo.svg", width: "24", height: "24", class: "text-gray-950") %>
+<% content_for :heading, "Create your account" %>
+<% if @person.errors.any? %>
+  <div id="error_explanation" class="px-3 py-2 mt-3 font-medium text-red-500 rounded-lg bg-red-50">
+    <ul>
+      <% @person.errors.full_messages.reverse.each do |error| # We want the User error first %>
+        <li><%= error.gsub(/Personable |credentials /, '').capitalize %><%# Cleanup "Personable password is ..." %></li>
+      <% end %>
+    </ul>
   </div>
-  <h2 class='text-3xl font-semibold'>Create your account</h2>
+<% end %>
+
+<% if Feature.google_authentication? %>
+  <%= button_to "Sign Up with Google", "/auth/google",
+    method: "post",
+    class: %|
+      text-white font-medium
+      bg-brand-blue dark:bg-gray-900
+      border border-white dark:border-gray-400
+      rounded-lg p-4 text-center
+      cursor-pointer
+      hover:opacity-95
+    |,
+    form_class: "flex flex-col space-y-4 w-80",
+    data: {
+      turbo: false,
+    }
+  %>
+  <span class="text-sm text-center <%= Feature.disabled?(:password_authentication) && 'hidden' %>">
+    - Or -
+  </span>
+<% end %>
+
+<% if Feature.password_authentication? %>
+  <%= form_with method: :post,
+        model: @person,
+        url: users_path,
+        class: "flex flex-col space-y-4 w-80",
+        data: {
+          controller: "transition",
+          transition_toggle_class: "max-h-0 max-h-40"
+        } do
+  |f| %>
```

Comment:
another indentation to pull back in. I won't flag all of them but it's worth a pass